### PR TITLE
add button to export children as csv

### LINKED ISSUE #252: Add export as CSV functionality for all resources
This needs to be fleshed out and broken out into different issues, but for now Ill list the page where we will want this...

- [ ] Pickup sheets
- [ ] Families
- [ ] Children

### PR DESCRIPTION
**Pull request by gabrielbaldao**
Resolves #252  

### Type of change

* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?

With a spec testing content type request, and i created a children_controller_spec to start with new tests about what was created.

### Screenshots

Before
antes

Now
agora

Csv file
csv

### CODE REVIEW

**Comment by renatamarques97** [comment]
this PR is duplicate #265

File: spec/controllers/children_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  let(:family)  { create :family}
+  let(:partner) { create :partner, name: "new partner", family_ids: [family.id] }
+  let!(:child) { create :child, family: family }
+  let(:user) { create :user, partner: partner }
+
+  before do
+    sign_in user
+  end
+  describe "GET #index" do
+    describe "when responding csv" do
```
Comment (renatamarques97) [inline spec/controllers/children_controller_spec.rb]: replace "describe" with "context"

File: spec/controllers/children_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      subject { get :index, format: :csv }
+
+      it "responds correct Content-Type" do
+        expect(subject.content_type).to eq "text/csv"
+      end
+    end
+  end
+end
```
Comment (renatamarques97) [inline spec/controllers/children_controller_spec.rb]: always end a file with a new line

**Comment by renatamarques97** [comment]
rubocop is failing, please take a look.

**Comment by benreyn** [comment]
Thanks @gabrielbaldao And @renatamarques97 !

**Pull request merged**